### PR TITLE
Updated ember version

### PR DESCRIPTION
**Pull request by suyashdeshpande**

### CODE REVIEW

File: app/templates/application.hbs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+{{!-- The following component displays Ember's default welcome message. --}}
+<WelcomePage />
```
Comment (abhishek97) [inline app/templates/application.hbs]: We don't really want this

Comment (suyashdeshpande) [reply app/templates/application.hbs]: @abhishek97  done

**Comment by suyashdeshpande** [comment]
@abhishek97 are there any more changes or is  it fine ?

**Comment by abhishek97** [comment]
part of updating dependencies is to fix warnings:

Screenshot 2019-05-26 at 2 17 07 PM

also update the `.nvmrc` as this doesn't build on the `8.9.1`

**Comment by suyashdeshpande** [comment]
@abhishek97  some of the warnings are related to ember-datetime-picker that it depends on ember-cli-babel 5.x

**Comment by abhishek97** [comment]
Yes, in such case we either update plugins or change it all together. For now ignore warning related to this plugin

**Comment by suyashdeshpande** [comment]
@abhishek97  pls check

**Comment by abhishek97** [comment]
@suyashdeshpande Not all warnings are because of ember-datetime-picker . Still need to fix the rest

**Comment by abhishek97** [comment]
Closing due to inactivity. Feel free to reopen.

**Pull request closed without merging**